streamlit_webrtc/frontend/src/index.tsx

```diff
@@ -1,29 +1,14 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import { createTheme, ThemeProvider } from "@material-ui/core/styles";
+import { ThemeProvider } from "./ThemeProvider";
 import CssBaseline from "@material-ui/core/CssBaseline";
 import { StreamlitProvider } from "streamlit-component-lib-react-hooks";
 import WebRtcStreamer from "./WebRtcStreamer";
 
-const theme = createTheme({
-  overrides: {
-    MuiCssBaseline: {
-      "@global": {
-        body: {
-          // Unset the background-color since <CssBaseLine /> applies the default Material Design background color
-          // (https://material-ui.com/components/css-baseline/#approach),
-          // which however does not match the Streamlit's background.
-          backgroundColor: "initial",
```

Comment:
memo:
With the latest version of streamlit-component-lib, the background color is forcefully injected, so there is no need to do it on developer's side.
https://github.com/streamlit/streamlit/blob/7aec5adce464ca73757d27c665df2c3ef2771df2/component-lib/src/streamlit.ts#L245